docs/io.cozy.contacts.md

```diff
@@ -84,6 +84,24 @@ Used to group contacts together. A group can obviously have multiple contacts, b
 - `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon
 - `metadata`: {object}
   - `version`: {integer} Used for migrations. Current version is **1**
+- `relationships`: {object} links between documents
+
+  - `accounts`: {object}
+    - `data`: {array} list of accounts
+      - `_id`: {string} id of the io.cozy.contacts.accounts document
+      - `_type`: {string} doctype "io.cozy.contacts.accounts"
+- `cozyMetaData`: {object}
+
+  - ...`cozyMetadata`: [Document metadata](./README.md#document-metadata)
+  - `sync`: {object}
+
+    - `ded4265b38c54b0683408c76d9ebd`: {object} id of the sourceContactsAccount
+
+      - `konnector`: {string} example : `"google"`
+      - `lastSync`: {date} (example: `"2018-10-19T10:58:37.025688+02:00"`)
```

Comment:
Is it updated everytime we sync even if we do not do any update?

Reply:
No, for me it's the same as https://github.com/cozy/cozy-doctypes/pull/84

Reply:
Maybe we should add a paragraph about the sync attribute to https://github.com/cozy/cozy-doctypes/blob/ea1b82f114383c92c02edd27c79e17fde75fe01d/docs/README.md#document-metadata ?

Reply:
Well #84 adds it in the relevant place. The whole sync section is not a standard cozyMetadata field, so I don't think it's the right place for it.